README.md

```diff
@@ -2,6 +2,86 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Development
+### Prerequisites
+* This repo uses `node` and `yarn`. 
```

Comment:
I use `npm` but that's not really important.

Reply:
Yeah I was thinking we could just say yarn in case we decide to introduce yarn workspaces or something similar in the future